NGitLab/Models/GroupQuery.cs

```diff
@@ -62,5 +63,56 @@ public class GroupQuery
         /// (optional)
         /// </summary>
         public AccessLevel? MinAccessLevel;
+
+        public GroupQueryScope Scope = GroupQueryScope.Projects;
```

Comment:
As it is required, let the user set it explicitly

```suggestion
        public GroupQueryScope? Scope;
```